src/formatOnSave/isPrettierInPackageJson.js

```diff
@@ -1,40 +0,0 @@
-// @flow
```

Comment:
I might have misunderstood, but isn't this the feature that lets the package use local Prettier installs instead of the packaged one? I thought it was useful for a lot of devs.

Reply:
This is the one that determines whether or not to run formatOnSave based on whether prettier is installed, if the user has chosen the option. The code that gets the local instance is still in (that's `getPrettierInstance`).